src/mlpack/core/optimizers/minibatch_sgd/minibatch_sgd_impl.hpp

```diff
@@ -92,6 +92,10 @@ double MiniBatchSGDType<
       Log::Info << "Mini-batch SGD: iteration " << i << ", objective "
           << overallObjective << "." << std::endl;
 
+      Log::Info << "Last objective" << lastObjective << std::endl;
+
+      Log::Info << "Gradient = " << arma::max(gradient) << std::endl;
```

Comment:
I understand you need that in order to debug the implementation, but we should remove that in the future.